src/torrent.cpp

```diff
@@ -7690,16 +7665,14 @@ namespace libtorrent
 			if (p.peer_info_struct() && p.peer_info_struct()->seed)
 				++seeds;
 
-			for (std::vector<pending_block>::const_iterator j = p.request_queue().begin()
-				, end(p.request_queue().end()); j != end; ++j)
+			for (auto const& j : p.request_queue())
```

Comment:
there are some conflicts here I believe, with a patch that I have in flight. and I'm afraid I intend to land mine first